moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -305,6 +305,13 @@ void MotionPlanningDisplay::reset()
   query_robot_goal_->setVisible(query_goal_state_property_->getBool());
 }
 
+void MotionPlanningDisplay::setName(const QString& name)
+{
+  BoolProperty::setName(name);
```

Comment:
@jrgnicho can I take that as your approval to merge+cherry-pick this as is?
Then I would directly do so.